src/functions/arrow.js

```diff
@@ -108,9 +116,11 @@ defineFunction({
         let lowerNode;
 
         if (group.body) {
-            const upperNode = mml.buildGroup(group.body, options);
+            let upperNode = mml.buildGroup(group.body, options);
+            upperNode = paddedNode(upperNode);
```

Comment:
I think function calls can be nest, as they're straightforward:
```js
const upperNode = paddedNode(mml.buildGroup(group.body, options));
```

Reply:
Done.